.github/workflows/ci.yaml

```diff
@@ -31,13 +31,16 @@ jobs:
           submodules: recursive
```

Comment:
*Arbitrary comment location*

Testing with Rainbow on mobile and wallet connect on desktop, I get a "Connection Failed" and "Connection Rejected" message on each immediately after scanning the QR code. This does not happen in Prod, where I'm able to connect without issue.

Reply:
It works with Metamask's wallet, and Rainbow works for me when I am running on my local computer, but I get the same error when I am using the deploy preview.